.eslintrc.js

```diff
@@ -1,139 +1,114 @@
-    'lodash-fp/prefer-composition-grouping': 'warn',
-    'lodash-fp/prefer-flat-map': 'warn',
-    'lodash-fp/prefer-get': 'warn',
-    'lodash-fp/prefer-identity': ['warn', { 'arrowFunctions': false }],
-    'lodash-fp/preferred-alias': 'warn',
-    'lodash-fp/use-fp': 'warn',
+    // TODO: ticket all of these to be enabled or explain why they are disabled
+    "react/forbid-prop-types": "off",
+    "react/prop-types": "off",
+    "react/sort-comp": "off",
+    "react/destructuring-assignment": "off",
+    "react-hooks/exhaustive-deps": "off",
+    "react/require-default-props": "off",
+    "react/static-property-placement": "off",
 
+    "import/no-cycle": "off",
+    "import/no-named-as-default": "off",
+    "import/no-extraneous-dependencies": "off",
+    "import/extensions": "off",
+    "import/no-relative-parent-imports": "off",
+    "import/named": "off",
+    "import/no-named-as-default-member": "off",
 
-    'react/default-props-match-prop-types': 'warn',
-    'react/no-children-prop': 'warn',
-    'react/no-unused-state': 'warn',
+    "@typescript-eslint/default-param-last": "off",
+    "@typescript-eslint/return-await": "off",
+    "@typescript-eslint/strict-boolean-expressions": "off", // TODO: should be 'warn',
+    "@typescript-eslint/naming-convention": "off",
+    "@typescript-eslint/no-shadow": "off",
+    "@typescript-eslint/no-use-before-define": "off",
+    "@typescript-eslint/no-unused-expressions": "off",
+    "@typescript-eslint/no-implied-eval": "off",
+    "@typescript-eslint/no-throw-literal": "off",
+    "@typescript-eslint/no-var-requires": "off",
+    "@typescript-eslint/no-empty-function": "off",
+    "@typescript-eslint/no-explicity-any": "off",
+    "@typescript-eslint/no-non-null-assertion": "off",
+    "@typescript-eslint/ban-ts-comment": "off",
+    "@typescript-eslint/ban-types": "off",
 
-    'react-hooks/rules-of-hooks': 'warn',
-    'react-hooks/exhaustive-deps': 'warn',
+    "no-case-declarations": "off",
+    "no-nested-ternary": "off",
+    "no-plusplus": "off",
+    "no-undef": "off",
+    "no-await-in-loop": "off",
+    "no-underscore-dangle": "off",
+    "no-param-reassign": "off",
+    "no-restricted-syntax": "off",
+    "no-void": "off",
+    "no-empty": "off",
+    "no-constant-condition": "off",
+    "no-promise-executor-return": "off",
+    "no-continue": "off",
+    "no-dupe-keys": "off",
+    "no-return-await": "off",
+    "no-console": "off",
+    "no-unused-expressions": "off",
 
-    'simple-import-sort/imports': 'warn',
+    "prefer-promise-reject-errors": "off",
+    "prefer-regex-literals": "off",
+    "prefer-destructuring": "off",
 
-    'import/named': 'warn',
-    'import/newline-after-import': ['warn', { 'count': 2 }],
-    'import/no-anonymous-default-export': ['warn', { 'allowObject': true }],
-    'import/no-internal-modules': ['warn', {
-      'forbid': ['src/components/common/*']
-    }],
-    'import/no-unresolved': 'warn',
+    "symbol-description": "off",
```

Comment:
Nit: can we sort rules alphabetically so that it's easier to check if a rule is configured?